core/controllers/collection_editor_test.py

```diff
@@ -77,8 +77,8 @@ def setUp(self) -> None:
 
     def test_access_collection_editor_page(self) -> None:
         """Test access to editor pages for the sample collection."""
-        whitelisted_usernames = [self.EDITOR_USERNAME]
-        self.set_collection_editors(whitelisted_usernames)
+        allowlisted_usernames = [self.EDITOR_USERNAME]
```

Comment:
Again, `allowed_usernames` sounds a bit better

Reply:
`allowed_usernames` sounds good to me too.